aws-lc-rs/src/ec/key_pair.rs

```diff
@@ -185,7 +183,7 @@ impl EcdsaKeyPair {
         alg: &'static EcdsaSigningAlgorithm,
         private_key: &[u8],
     ) -> Result<Self, KeyRejected> {
-        let evp_pkey = encoding::unmarshal_der_to_private_key(private_key, alg.id.nid())?;
+        let evp_pkey = ec::encoding::rfc5915::parse_rfc5915_private_key(private_key, alg.id.nid())?;
```

Comment:
Maybe remove the slight name stuttering, `rfc5915` is already in the module name, so I would remove it from the function name.

Reply:
I'm conflicted about removing the "rfc5915" from the function name as I like that it helps prevent future refactorings from potentially confusing two different functions both named `parse_private_key`. In the latest commit, I did some cleanup to remove the redundant use of "rfc5915" name in the logic here. Let's discuss this more on the next commit.